modules/hal_nordic/nrfx/nrfx_config.h

```diff
@@ -1136,6 +1136,8 @@
     #include <nrfx_config_nrf54l05_application.h>
 #elif defined(NRF54L05_XXAA) && defined(NRF_FLPR)
     #include <nrfx_config_nrf54l05_flpr.h>
+#elif (defined(NRF54L09_XXAA) | defined(NRF54L09_ENGA_XXAA)) && defined(NRF_APPLICATION)
```

Comment:
What's the point in checking the `NRF54L09_XXAA` symbol here? It doesn't seem to be defined/supported anywhere.

Reply:
Removed the `NRF54L09_XXAA` symbol.